src/callbacks_step/amr_dg2d.jl

```diff
@@ -383,11 +383,6 @@ end
 function adapt!(u_ode::AbstractVector, adaptor, mesh::T8codeMesh{2}, equations,
                 dg::DGSEM, cache, difference)
 
-    # Return early if there is nothing to do.
-    if !any(difference .!= 0)
-        return nothing
-    end
-
```

Comment:
Why?

Reply:
I reintroduced this if-clause and call `reinitialize_containers` in case of MPI mode.